pysages/colvars/patterns.py

```diff
@@ -42,6 +44,9 @@ def __init__(
         centre_j_id,
         standard_deviation,
         mesh_size,
+        number_of_added_sites=0,
```

Comment:
If the continuous version of the LoM is to be used, the additional atoms should already be added at the end of the `reference` (list of coordinates of the reference structure). With `number_of_added_atoms` we specify how many were included.